assets/vue/components/migrate-notice.vue

```diff
@@ -46,30 +46,49 @@
 				dismissed: false,
 			}
 		},
-		computed: {
-			migrationData: function () {
-				return this.$store.state.sitesData.migrate_data
+		props: {
+			migration_data: {
+				default: {},
+				type: Object,
+				required: true,
 			},
 		},
 		methods: {
+			setupMigrationData: function () {
```

Comment:
This is duplicated code - same as `stie-item.vue`. They can be put into a `common.js` file and use parameters as they seem to do the same thing. Then you can use the imported method here.